src/daemon/emitter.cc

```diff
@@ -68,6 +68,30 @@ inline bool GenerateSource(const base::proto::Local* WEAK_PTR message,
   return true;
 }
 
+inline bool ReadExtraFiles(const base::proto::Local* WEAK_PTR message,
```

Comment:
Why not use `ReadExtraFiles()` from compilation_daemon.cc ? - I don't see the difference.